jsr.jsonc

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@bureaudouble-forks/react-remove-scroll",
```

Comment:
what is bureaudouble-forks?

Reply:
not trying to be mean, I'm following multiple repo looking for new releases and many of them received similar PRs w/ same namespace and user, it does seem odd... but have no idea how deno actually works... you can see his other PRs: https://github.com/search?q=bureaudouble&type=pullrequests

Reply:
https://github.com/joe-bell/cva/pull/278#issuecomment-2055497351

Reply:
Thanks @maiconcarraro, I am really inclined to follow @joe-bell's example and close these multiple PRs (opened one day for multiple packages I maintain) without any explanation or link to get extra information from. And I cannot maintain what I do not understand in the details.

Hopefully, I am aware of what JSR is, but
- `bureaudouble`? 🤷‍♂️ 
- "zero major versions?" 🤷‍♂️ 
- "imports" 🤷‍♂️ 
- ESM support? 🤷‍♂️ 
- `// @deno-types=` for _some_ imports? 🤷‍♂️ 

What actually the fuck, and what actually the goal?

Reply:
Straight up spam IMO

Reply:
I only found out after doing some further reading; `bureaudouble` is actually his personal account scope

This just feels like straight up spam to me

Reply:
I have made the required changes so that the package could be delivered in jsr.io, I thought it would be in the interest of the community to make it open, thus the PR.

I'm open to whatever feedback that is constructive and I'm kind of sad about what you've written.

About the bureaudouble thing, it's the scope I use in jsr, it's up to you to change it, Ive made those changes for my work. But yes the PR still needs some works, up to you to use it or discuss improvements.

Anyway